### PR TITLE
GitKraken latest (switch from zip to DMG)

### PR DESCRIPTION
**Pull request by crahan**
After making all changes to the cask:

- [X] `brew cask audit --download {{cask_file}}` is error-free.
- [X] `brew cask style --fix {{cask_file}}` reports no offenses.
- [X] The commit message includes the cask’s name and version.
- [X] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

This is a pull request as requested at https://github.com/Homebrew/homebrew-cask/issues/63563. The original .zip file which is downloaded from https://release.gitkraken.com/darwin/GitKraken-v5.0.4.zip contains a GitKraken application which is over 500MB. The .dmg version downloaded from https://release.gitkraken.com/darwin/installGitKraken.dmg (which is the location used when directly downloading from the GitKraken website) contains an application that's only half the size. The .dmg version also does not trigger an unidentified developer warning when launched.

Unfortunately, I did have to change the versioning from an actual version number to `:latest` which does not check the SHA256 hash, doesn't support auto_update, and doesn't use appcast (per the `brew cask audit --download` output).

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

**Comment by vitorgalvao** [comment]
> I did have to change the versioning from an actual version number to `:latest` which does not check the SHA256 hash, doesn't support auto_update, and doesn't use appcast

Those aren’t mutually exclusive. Revert everything except for the `url` change, and update the `sha256`. We’ll continue from there.

**Comment by crahan** [comment]
Changes have been applied as requested.

Using this configuration, even though the installer package (DMG) does not indicate a particular version number, the SHA256-check will warn of any updates or changes to the installGitKraken.dmg disk image and require someone to update the version number and SHA256 in `Casks/gitkraken.rb`?

**Comment by vitorgalvao** [comment]
> the SHA256-check will warn of any updates or changes to the installGitKraken.dmg disk image and require someone to update the version number and SHA256 in `Casks/gitkraken.rb`?

Yes. The `appcast` will help identify updates faster.

For anyone that lands here later on, quick recap of the reasons for the change:

* Zip version is double (185MB) the size of the DMG (91MB).
* Zip version is unsigned.

**Review by vitorgalvao — APPROVED** [review]

File: Casks/gitkraken.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 cask 'gitkraken' do
   version '5.0.4'
-  sha256 '16bdd399bcd75ac4d48e639eecfd0b583efab0482882757e241987d85e0b9b41'
+  sha256 '0d6df7e7c686af55a2e2033729ad5bfa22cb1e36090cc999eebd10bf9fc2dcdd'
 
-  url "https://release.gitkraken.com/darwin/GitKraken-v#{version}.zip"
+  url "https://release.gitkraken.com/darwin/installGitKraken.dmg"
```
Comment (vitorgalvao) [inline Casks/gitkraken.rb]: These need to be changed to single quotes.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/gitkraken.rb (hunk 1 above)
Comment (crahan) [reply Casks/gitkraken.rb]: Fixed.

**Comment by vitorgalvao** [comment]
Thank you, @crahan.

**Pull request merged**